src/ui/components/ObjectLabel.vue

```diff
@@ -21,6 +21,7 @@
 -->
 <template>
   <a
+    ref="root"
```

Comment:
Why do we need this again?

Reply:
This works in conjunction with context-menu-gesture mixin